bin/reth/src/db/mod.rs

```diff
@@ -232,6 +270,15 @@ impl<'a, DB: Database> DbTool<'a, DB> {
             .map_err(|e| eyre::eyre!(e))
     }
 
+    fn get<T:Table>(&mut self, idx: usize) {
```

Comment:
```suggestion
    fn get<T:Table>(&mut self, idx: T::Key) -> T::VAlue {
```

This should be the table's key and you should just do `self.db.view(|tx| tx.get(key))` as far as i can tell